Products/CMFPlone/patterns/__init__.py

```diff
@@ -242,6 +241,8 @@ def tinymce(self):
 
         image_types = settings.image_objects or []
         folder_types = settings.contains_objects or []
+
+        site_path = generator.portal_url.replace(self.request['SERVER_URL'], '')
```

Comment:
Maybe off-topic since I did not look at how ``SERVER_URL`` changes with rewrite rules. But please do not use a regex here. I consider regular expressions as write-only code which is generally bad. Use ``urlparse`` instead:

```python
>>> from urlparse import urlparse
>>> o = urlparse('webdav://domainname.com/abc/xyz')
>>> o.path
'/abc/xyz'
```